drivers/segger/note_sysview.c

```diff
@@ -310,6 +399,42 @@ void sched_note_irqhandler(int irq, FAR void *handler, bool enter)
 }
 #endif
 
+#ifdef CONFIG_SCHED_INSTRUMENTATION_SYSCALL
+void sched_note_syscall_enter(int nr, int argc, ...)
+{
+  if (sysview_isenabled_syscall(nr) == 0)
+    {
+      return;
+    }
+
+  if (NOTE_FILTER_SYSCALLMASK_ISSET(nr, &g_sysview.syscall_active) == 0)
```

Comment:
need nr - NOTE_FILTER_SYSCALLMASK_ISSET in all NOTE_FILTER_SYSCALLMASK_xxx

Reply:
Use the Syscall "0" to identify whether the syscall is enabled, if the host tool is closed abnormally, use this bit to clear the active set.